Add missing word to ripplestate doc (#1404)
XRPLF/xrpl-dev-portal: content/references/protocol-reference/ledger-data/ledger-object-types/ripplestate.md

|-----------------|-----------|---------------|-------------|
| `LedgerEntryType` | String    | UInt16 | The value `0x0072`, mapped to the string `RippleState`, indicates that this object is a RippleState object. |
| `Flags`           | Number    | UInt32 | A bit-map of boolean options enabled for this object. |

> | `Balance`         | Object    | Amount | The balance of the trust line, from the perspective of the low account. A negative balance indicates that the low account has given tokens to the high account. The issuer in this is always set to the neutral value [ACCOUNT_ONE](accounts.html#special-addresses). |

Reviewer: ```suggestion
| `Balance`         | Object    | Amount | The balance of the trust line, from the perspective of the low account. A negative balance indicates that the high account holds tokens issued by the low account. The issuer in this object is always set to the neutral value [ACCOUNT_ONE](accounts.html#special-addresses). |
```
